index.bs

```diff
@@ -1383,7 +1407,7 @@ attributes.
     dictionary AuthenticatorSelectionCriteria {
         AuthenticatorAttachment      authenticatorAttachment;
         boolean                      requireResidentKey = false;
-        boolean                      requireUserVerification = false;
+        UserVerificationRequirement  requireUserVerification = "preferred";
```

Comment:
we may have an impedance mis-match here with the default value of "preferred".  CTAP has UV default to `false` on its `authenticatorMakeCredential` method.

Reply:
So should we change this default back to `"discouraged"`?

Reply:
I think we need consensus from amongst implementors as to requireUserVerification's default value.  cc: @balfanz  @battre @jcjones  @AngeloKai

Reply:
The advantage we see in having it default to `"discouraged"` is that implementations don't default to a mechanism that will provide more information than they need. It also might keep the common case fast, for a hypothetical authenticator that requires different actions depending on `UV`'s state.

That said, I think there are enough U2F devices in the field that implementors realistically will have to deal with no-UV information for a long time to come, so I won't object to keeping this `"preferred"`.